engine-nextgen/src/main/kotlin/graphql/nadel/enginekt/plan/NadelExecutionPlanFactory.kt

```diff
@@ -73,11 +74,12 @@ internal class NadelExecutionPlanFactory(
                 executionBlueprint,
                 transforms = listOfTransforms(
                     *transforms.toTypedArray(),
```

Comment:
Unrelated to your changes, but I really hate how you can't just spread a collection. So dumb.